src/functionalTest/groovy/com/autonomousapps/jvm/projects/IncludedBuildProject.groovy

```diff
@@ -68,7 +68,7 @@ final class IncludedBuildProject extends AbstractProject {
   final Set<ProjectAdvice> expectedBuildHealth = [
     projectAdviceForDependencies(':', [
       Advice.ofRemove(
-        includedBuildCoordinates('second:second-build', '1.0', projectCoordinates(':second-build')),
+        includedBuildCoordinates('second:second-build', projectCoordinates(':')),
```

Comment:
The project coordinate `project(':second-build')` does not exist in the Gradle-sense. There is an included build in which the root project would be referred to as `project(':')`.